src/formula/function.cpp

```diff
@@ -448,6 +449,29 @@ DEFINE_WFL_FUNCTION(replace, 3, 4)
 	return variant(result.replace(offset, std::string::npos, replacement));
 }
 
+DEFINE_WFL_FUNCTION(replace_all, 3, 3)
+{
+	std::string result = args()[0]->evaluate(variables, fdb).as_string();
+	std::string needle = args()[1]->evaluate(variables, fdb).as_string();
+	std::string replacement = args().back()->evaluate(variables, fdb).as_string();
+	boost::replace_all(result, needle, replacement);
```

Comment:
The documentation for boost::replace_all doesn't answer my questions - does this correctly handle reallocating the string if replacement is longer than needle?

Reply:
I can't tell either, but I would assume it would ultimately boil down to `string::replace` at some level, or `string::insert` at worst.

I did test it with a replacement longer than the needle, and it looks like it works.